### PR TITLE
CDAP-17402 Upgrade debezium to 1.3.1.Final version.

### PR DESCRIPTION
**Pull request by sagarkapare**
https://cdap.atlassian.net/browse/CDAP-17402

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -58,7 +58,7 @@
     <jee.version>7</jee.version>
     <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
     <cdap.version>6.4.0-SNAPSHOT</cdap.version>
-    <debezium.version>0.9.5.Final</debezium.version>
+    <debezium.version>1.3.1.Final</debezium.version>
```
Comment (seanzhougoogle) [inline pom.xml]: Not related to this PR, but why are we defining the verion of debezium at delta repo ? That is source plugin specific , right ?

Comment (sagarkapare) [reply pom.xml]: yeah thats right. Let's remove it.

Comment (sagarkapare) [reply pom.xml]: Its unused

**Review by seanzhougoogle — APPROVED** [review]

**Comment by sagarkapare** [comment]
Build is green here - https://builds.cask.co/browse/DEL-BT85-4. Squashing and merging.

**Pull request merged**